tests/reporter/kubernetes.go

```diff
@@ -1115,6 +1174,37 @@ func (r *KubernetesReporter) executePriviledgedVirtLauncherCommands(virtCli kube
 	}
 }
 
+func (r *KubernetesReporter) executeCloudInitCommands(vmi v12.VirtualMachineInstance, path string, vmiType string) {
```

Comment:
this file could use better names for functions, but I guess it is consistent with existing usage and it is best handled in a separate PR.

Reply:
Yep, I am planning to post small refactoring PR for this file soon, we can do it then :slightly_smiling_face: